test.js

```diff
@@ -0,0 +1,234 @@
+
+const nbCommonFields = 2;
+
+const MS_PER_MINUTE = 60000;
+
+describe('eventToBuild', () => {
+  it('should transform a base64 build to an object', () => {
+    let build = lib.eventToBuild(base64Build);
+    should.exist(build.projectId);
+    build.projectId.should.equal('node-example-gke');
+  });
+});
+
+describe('createSlackMessage', () => {
+  it('should create a slack message', () => {
+    let build = {
+      id: 'build-id',
+      logUrl: 'https://logurl.com',
+      status: 'SUCCESS',
+      finishTime: '2017-03-19T00:08:12.220502Z',
+    };
+    let message = lib.createSlackMessage(build);
+
+    message.text.should.equal("Build `build-id` finished");
+    should.exist(message.attachments);
+    message.attachments.should.have.length(1);
+    let attachment = message.attachments[0];
+    attachment.title_link.should.equal(build.logUrl);
+    attachment.ts.should.equal(1489882092);
+    attachment.fields.should.have.length(nbCommonFields);
+    attachment.fields[0].value.should.equal(build.status);
+  });
+
+  it('should include the build duration as a field', () => {
+    let now = Date.now();
+    let deltaInMinutes = 11;
+    let build = {
+      id: 'build-id',
+      logUrl: 'https://logurl.com',
+      status: 'SUCCESS',
+      startTime: new Date(now - deltaInMinutes*MS_PER_MINUTE),
+      finishTime: now,
+    };
+    let message = lib.createSlackMessage(build);
+
+    let attachment = message.attachments[0];
+    attachment.fields[1].value.should.equal(deltaInMinutes + ' minutes');
+  });
+
+  it('should create a slack message with images', () => {
+    let build = {
+      id: 'build-id',
+      logUrl: 'https://logurl.com',
+      status: 'SUCCESS',
+      finishTime: Date.now(),
+      images: ['image-1', 'image-2'],
+    };
+    let message = lib.createSlackMessage(build);
+
+    let attachment = message.attachments[0];
+    attachment.fields.should.have.length(nbCommonFields + build.images.length);
+    attachment.fields[nbCommonFields].value.should.equal('image-1');
+    attachment.fields[nbCommonFields+1].value.should.equal('image-2');
+  });
+
+  it('should use the right color depending on the status', () => {
+    let build = {
+      id: 'build-id',
+      finishTime: Date.now(),
+    };
+    let testCases = [
+      {
+        status: 'QUEUED',
+        want: '#4285F4',
+      },
+      {
+        status: 'WORKING',
+        want: '#4285F4',
+      },
+      {
+        status: 'SUCCESS',
+        want: '#34A853',
+      },
+      {
+        status: 'FAILURE',
+        want: '#EA4335',
+      },
+      {
+        status: 'INTERNAL_ERROR',
+        want: '#EA4335',
+      },
+      {
+        status: 'TIMEOUT',
+        want: '#FBBC05',
+      },
+    ];
+    testCases.forEach(function(tc) {
+      build.status = tc.status;
+      let message = lib.createSlackMessage(build);
+      message.attachments[0].color.should.equal(tc.want, tc.status);
+    });
+  });
+});
+
+function cleanConfig(callback) {
```

Comment:
I'd probably move `cleanConfig` inside the next describe section since it's only used there.